_posts/2019-02-01-4th-workshop-announcement.md

```diff
@@ -0,0 +1,96 @@
+
+### Agenda
+
+The the first part of the workshop (17-19th of June) has two main objectives:
+
+1. **State of OGGM** : the first sessions will be spent with presentations from the
+participants about recent developments and results.
+
+2. **Future developments and visions for OGGM** : the rest of the sessions
+will be dedicated to general discussions and brainstorming about the future and 
+goals of the OGGM project.
+
+This workshop will be followed by two days of training and coding session 
+in smaller groups at the IGE in Grenoble:
+
+- **Beginner tutorials**: these sessions are for current and future OGGM users to
+  get to know the model internals, and learn how to set-up and customize a model run.
+- **Code sprints**: we will address specific coding challenges
+  together, in the form of a [programming sprint](https://oggm.org/2018/10/16/hack-day/).
+- **Q&A**: we will leave time open to discuss actual problems in the model itself,
+  and try to develop strategies to solve them.
+
+This second part is targetting current and future model *users*, or 
+maybe just people interested in the model internals. 
+It will be rather technical and could be skipped if you do not plan to
+use the model yourself.
+
+### Who can participate ?
+
+Anyone interested in the model, or in glaciological modeling
+in general! In particular, we would like to encourage potential users and
+developers to join us to get to know each other and to gather first-hand
+information about the model internals, as well as PhD students and young scientists.
+Note that in case of high interest we might have to limit the number of participants.
+
+### Workshop fee
+
+**For the workshop part**, the hotel accomodation (night & breakfeast)
+is €90 in a single room and €56 in a twin room. The prices are per night per person.
+The conference fees are €240 covering all other organization costs
+(including all meals).
+
+**For the training part** there are no fees, but attendants should take 
+care of their own accomodation and meals.
+
+### Registration
+
+If you wish to participate to the workshop or if you have questions regarding
+the organization please send us an e-mail at _info [at] oggm.org_
+**before March 31st, 2019**.
+
+### Organisators
+
+[Nicolas Champollion](https://geographie.uni-bremen.de/index.php?option=com_jresearch&view=member&task=show&id=87), [Fabien Maussion](http://fabienmaussion.info/) and [Ben Marzeion](http://marzeion.info/).
+
+### Past workshops
+
+- <u> <a href="{{ site.url }}/2016/02/11/1st-oggm-worshop-summary/"> 1st OGGM workshop </a> </u>
+- <u> <a href="{{ site.url }}/2017/04/03/2nd-oggm-worshop-summary/"> 2nd OGGM workshop </a> </u>
+- <u> <a href="{{ site.url }}/2018/06/29/3nd-oggm-worshop-summary/"> 3rd OGGM workshop </a> </u>
+
+### Announcement
+
+[Workshop Announcement](https://docs.google.com/document/d/1j6nFeEDh777P-97F4ToTA4KJ4fOBiuXGG73nyCjaZdg/edit?usp=sharing)
```

Comment:
@nchampollion are you sure this is necessary? The blog post should be enough right?

Reply:
I would use an official PDF program once we have all the talks, etc.

Reply:
How can Ben gives some comments as well ? He asked me and I was not sure about the response.

Probably yes that is not necessary. It was done if people wants to share document announcements to other (instead of blog-post) and to have this document ready if we want to send to the participants ... maybe too redundant.

Reply:
Ok !

Reply:
> maybe too redundant

yes - two documents are two times more chance to make a mistake :D. Maybe we should include Ben's omments in this post if appropriate

Reply:
Yes, I do that later on this afternoon and produce a final version for validation